stl/inc/format

```diff
@@ -1806,12 +1806,6 @@ class _Fmt_iterator_buffer<back_insert_iterator<_Container>, _RANGES range_value
 private:
     _Container& _Cont;
 
-    struct _Accessor : back_insert_iterator<_Container> {
-        explicit _Accessor(back_insert_iterator<_Container> _Iter) : back_insert_iterator<_Container>(_Iter) {}
-
-        using back_insert_iterator<_Container>::container;
-    };
-
     void _Grow(size_t _Capacity) final {
         _Cont.resize(_Capacity);

```

Comment:
Sidenote: this call to resize is not guarded by any concept, so a theorical container with `push_back` but not `resize` could cause this to emit compilation errors.